locales/en.json

```diff
@@ -376,6 +376,16 @@
   "errors.symptoms_occur_required": "Symptoms occur is required",
   "Thank you!": "Thank you!",
   "skip_section": "Skip this section",
+  "conditions.medical_conditions": "Medical conditions",
+  "conditions.icd_code": "ICD-9-CM code: ",
+  "conditions.symptoms_began": "Symptoms began: ",
+  "conditions.remove_condition": "Remove condition",
+  "conditions.add_a_condition": "Add another condition",
+  "conditions.next_medications": "Next: Medications",
+  "Add a medication": "Add a medication",
```

Comment:
I'm guessing this key is unused and looks to be a duplicate of line 388 below

Reply:
removed